apps/api-users/src/app/modules/user/user.service.spec.ts

```diff
@@ -61,6 +61,18 @@ describe('UserService', () => {
     })
   })
 
+  describe('getByUserId', () => {
+    beforeEach(() => {
+      ;(service.db as DeepMockProxy<Database>).query.mockReturnValue(
```

Comment:
does it work without the ;?
```suggestion
      (service.db as DeepMockProxy<Database>).query.mockReturnValue(
```

Reply:
The semi is created by prettier, won't pass without it